flink/src/main/java/org/apache/iceberg/flink/RowTaskWriterFactory.java

```diff
@@ -120,11 +121,12 @@ private FlinkFileAppenderFactory(Schema schema, Map<String, String> props) {
     @Override
     public FileAppender<Row> newAppender(OutputFile outputFile, FileFormat format) {
       MetricsConfig metricsConfig = MetricsConfig.fromProperties(props);
+      LogicalType logicalType = FlinkSchemaUtil.convert(schema);
```

Comment:
This should have simiar issue to the [comment](https://github.com/apache/iceberg/pull/1320#discussion_r468808965), which will break the [unit test](https://github.com/apache/iceberg/pull/1320/commits/44da7c33f883d472801b1195d25b54405846d4c5#diff-6820fc22b4e5cbfa4a1c029bf5c8c789R253). If we rebase the master once  https://github.com/apache/iceberg/pull/1320 get merged, then it should have no problem.

Reply:
BTW,  we may also need to add the `parquet` into the parameterized unit tests, such as `TestIcebergStreamWriter` & `TestTaskWriters`.

Reply:
Agreed,  will take a look when these PRs get in.